react-game/src/components/AllTrials.js

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import StartButton from './Buttons/StartButton';
+import classes from './AllTrials.module.css';
+import TrialRow from './TrialRow/TrialRow';
+
+function AllTrials() {
+
+    //STATES & HOOKS-------------------------------------------------------------------
+    const [secretCode, setSecretCode] = useState("0000");
+    const [selectedIndex, setSelectedIndex] = useState(0); //this state holds the current enabled row  
+    const [gameStatus, setGameStatus] = useState(0); // 0 - means the game has not started yet 
+    // 1 - means the game is active
+    // 2 - means the game finished successfully
+    // 3 - means the game finished with a failure 
```

Comment:
please create const (outside this component) contains these states  and use it inside this component
example: 
const GAME_STATUS = {
    NOT_STARTED = 0,
    ACTIVE = 1,
    WON = 2,
    LOST = 3
}

Reply:
done